Tools-Override/CloudTest.targets

```diff
@@ -1,431 +1,173 @@
-
-  <Import Project="$(MSBuildThisFileDirectory)CloudTest.Perf.targets" Condition="'$(Performance)' == 'true'" />
 
-  <!-- main entrypoint -->
-  <Target Name="CloudBuild"
-          AfterTargets="Build"
-          DependsOnTargets="VerifyInputs;PreCloudBuild;CreateTestListJson;UploadContent" />
+    <!-- Local path for storing submission-related files -->
+    <HelixLogFolder>$(ArchivesRoot)</HelixLogFolder>
 
-  <Target Name="VerifyInputs">
-    <!-- verify all required properties have been specified -->
-    <Error Condition="'$(Creator)' == '' and '$(HelixApiAccessKey)' == ''" Text="Missing required property Creator." />
-    <Error Condition="'$(TargetQueue)' == ''" Text="Missing required property TargetQueue." />
-    <Error Condition="'$(TestProduct)' == ''" Text="Missing required property TestProduct." />
-    <Error Condition="'$(BuildMoniker)' == ''" Text="Missing required property BuildMoniker." />
-    <Error Condition="'$(Branch)' == ''" Text="Missing required property Branch." />
-
-    <Error Condition="'$(CloudDropAccountName)' == ''" Text="Missing required property CloudDropAccountName." />
-    <Error Condition="'$(CloudResultsAccountName)' == ''" Text="Missing required property CloudResultsAccountName." />
-    <Error Condition="'$(CloudDropAccessToken)' == ''" Text="Missing required property CloudDropAccessToken." />
-    <Error Condition="'$(CloudResultsAccessToken)' == ''" Text="Missing required property CloudResultsAccessToken." />
-
-    <Warning Condition="'$(EventHubPath)' != '' or '$(EventHubSharedAccessKey)' != '' or '$(EventHubSharedAccessKeyName)' != ''" Text="EventHubPath and EventHubAccessKeys are ignored, use HelixApiAccessKey instead" />
-    <Error Condition="'$(SkipNotifyEvent)' != 'true' and '$(HelixApiAccessKey)' == ''" Text="HelixApiAccessKey must be set to start Helix jobs" />
+    <!-- Properties used for submission by CloudTest.Helix.Targets-->
+    <BuildMoniker>$(CurrentDate)</BuildMoniker>
+    <BuildMoniker Condition="'$(IsOfficial)'=='true'">$(OfficialBuildId)</BuildMoniker>
+    <!-- TODO:  This requires some specific understanding of Helix, we should try to generalize this into properties.  -->
+    <HelixJobProperties>{ &quot;architecture&quot; : &quot;$(ArchGroup)&quot;, &quot;configuration&quot;: &quot;$(ConfigurationGroup)&quot;, &quot;operatingSystem&quot; : &quot;$(TargetOS)&quot; }</HelixJobProperties>
+  </PropertyGroup>
 
-    <!-- gather the test archives for this build -->
+  <Target Name="CoreFXPreCloudBuild" >
+    <!-- Put anything you want to do before submitting the general purpose helix job here.
+         Helix requires properly formatted work items and accessible zip files / others to upload, which we'll construct here. -->
+    
+    <!-- Gather the test archives for this build 
+         If you change binplacing behavior and official runs break, 
+         this is a good place to check.-->
     <ItemGroup>
       <OSPlatformTestArchives Include="$(TestArchivesRoot)**/*.zip" >
```

Comment:
We should completely eliminate the need to find all these different OSConfiguration output paths. Instead we should change https://github.com/dotnet/corefx/blob/5b01b1942f56a249623e9bfb31de35eef74f1b4f/Tools-Override/publishtest.targets#L96 to always archive tests to a single root folder. Essentially moving the specific OS configuration stuff to the name of the zip instead of the directory name. That make this logic as simple as $(TestArchivesRoot)*.zip.